src/lib/blockchain/metamask.ts

```diff
@@ -167,6 +167,72 @@ export class Metamask {
     return MESSAGE;
   }
 
+  private getDefaultChainParam(){
+    if (POLYGON_TESTNET_CHAIN_ID === 137) {
+      return {
+        chainId: `0x${Number(POLYGON_TESTNET_CHAIN_ID).toString(16)}`,
```

Comment:
We can rename this constant to `POLYGON_CHAIN_ID` since it will represent the testnet and mainnet.